cert/go.mod

```diff
@@ -1,8 +1,11 @@
 module github.com/lightningnetwork/lnd/cert
 
-go 1.16
+go 1.19
```

Comment:
We'll need to push up tags for all modules and bump their version in the main `go.mod`, but I guess that's always nice to make sure we include all the latest changes.